include/nuttx/sched.h

```diff
@@ -244,7 +244,7 @@
 #  define this_cpu()                 (0)
 #endif
 
-#define running_regs()               ((void *)(g_running_tasks[this_cpu()]->xcp.regs))
+#define running_regs()               ((void **)(g_running_tasks[this_cpu()]->xcp.regs))
```

Comment:
> Should this be `(FAR void *)(g_running_tasks[this_cpu()]->xcp.regs)` or `(FAR void *)g_running_tasks[this_cpu()]->xcp.regs` ?
> 
> Can you try build with -Wpointer-arith just in case

will also report compile error